### PR TITLE
[3.4] bug-fix/issue-#8294

### PR DESCRIPTION
**Pull request by gnusi**
Please note that for legal reasons we require you to sign the 
[Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Comment by gnusi** [comment]
https://github.com/arangodb/arangodb/issues/8294

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/

File: arangod/Aql/Condition.cpp [hunk 1]
```diff
@@ -1032,7 +1129,7 @@ void Condition::optimize(ExecutionPlan* plan) {
           // end of IN-merging
 
           // Results are -1, 0, 1, move to 0, 1, 2 for the lookup:
-          ConditionPartCompareResult res = ConditionPart::ResultsTable
+          ConditionPartCompareResult res = resultsTable
```
Comment (graetzer) [inline arangod/Aql/Condition.cpp]: why is this only used here ?

Comment (gnusi) [reply arangod/Aql/Condition.cpp]: good catch, I missed it

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/4930/

**Pull request merged**